### PR TITLE
Updated with HOTHX info

### LINKED ISSUE #359: Update HOTH event info
Several things need to be updated or added
- hoth banner in highlights.js needs to be updated when the design asset comes out
- Hoth event card needs to be added when data is known about when and where hoth will be held
- RSVP link needs to be added to hoth event highlight

### PR DESCRIPTION
**Pull request by nathanzzhang**
# Changes

Updated HOTHX banner in highlights, updated date and location of HOTHX in events.

## Type of change

Please delete options that are not relevant.

- [x] Content Update (non-breaking change which updates the content of the website)

# Related Issues

Fixes #359

### CODE REVIEW

File: src/data/events/highlights.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ const highlightedEvents = [
 	{
 		name: 'Hack on the Hill X',
 		description: HOTHXDescription,
-		imgFilePath: 'event/2023w-hoth-x-banner.png', // TODO: update banner
+		imgFilePath: 'event/2023w-hoth-x-banner.png',
```
Comment (katelynsyu) [inline src/data/events/highlights.js:28]: You should be able to create another button here to say “sign up” (I think it’s `rsvp` but I could be wrong) that allows you to link the rsvp form

**Review by katelynsyu — CHANGES_REQUESTED** [review]

File: src/data/events/highlights.js (hunk 1 above)
Comment (nathanzzhang) [reply src/data/events/highlights.js:28]: it seems like the highlights don't have the capability to have 2 buttons. the rsvp link is in the event card, however. let me know if I'm wrong LOL

Comment (katelynsyu) [reply src/data/events/highlights.js:28]: Huh. I must be tripping then. In that case can you change the text on the button from 'More Info' to 'Sign Up'? Ty :)

**Review by katelynsyu — APPROVED** [review]
Perfection. Thank you!

**Pull request merged**